docs/source/getting_started.rst

```diff
@@ -36,3 +36,11 @@ Installing FESTIM
 FESTIM can then be installed using pip::
 
     pip install FESTIM
+
+Previous version of FESTIM can then be installed using pip::
+
+    pip install FESTIM==version
+
+with the desired version number.  For example::
```

Comment:
```suggestion
with the desired version tag.  For example::
```

Reply:
Yes! These great suggestions improve clarity of the documentation, I agree with you

Reply:
You can commit them directly from this PR